js/index.js

```diff
@@ -201,6 +201,14 @@ function eraseAlert() {
   message.innerHTML = '';
 }
 
+function changeInput() {
+  const inputName = document.getElementById('survey-form-name-input').value;
+  const inputEmail = document.getElementById('survey-form-email-input').value;
+  const inputMessage = document.getElementById('survey-form-message-input').value;
+  const dataStorage = { name: inputName, email: inputEmail, message: inputMessage };
```

Comment:
- [ ] Nice implementation of object for you inputs, Kindly make sure we can send the input value and receive the value from the local storage. you can write your object as follow.👇

```
const dataStorage = {
    name: InputName.value,
    email: InputEmail.value,
    message: inputMessage.value,
};
```